GVFS/GVFS.Notifications/VFSForGit.Mac/StatusMenuItem/IPC/MessageListener.h

```diff
@@ -0,0 +1,15 @@
+#import <Foundation/Foundation.h>
+
+typedef void (^NewMessageCallback) (NSDictionary *_Nonnull messageInfo);
```

Comment:
Recent versions of Xcode insert the following markers in header files, to default to _Nonnull for all parameters. This class looks like would benefit from that instead of explicitly adding _Nonnull decorators everywhere.
NS_ASSUME_NONNULL_BEGIN
…your code...
NS_ASSUME_NONNULL_END